src/exchange/billing/account-renew/billing-account-renew.html

```diff
@@ -0,0 +1,136 @@
+<div class="px-5 pt-5" data-ui-view>
+    <oui-back-button data-on-click="$ctrl.goBack()">
+        <span data-translate="exchange_update_billing_action_title"></span>
+    </oui-back-button>
+
+    <oui-stepper
+        data-on-finish="submit()">
```

Comment:
```suggestion
        data-on-finish="$ctrl.submit()">
```
?

Reply:
It works since it is exposed on the $scope, but it could be refactored :)